addons/l10n_ar/models/account_move.py

```diff
@@ -132,14 +132,27 @@ def _onchange_partner_journal(self):
             res_code = rec.partner_id.l10n_ar_afip_responsibility_type_id.code
             domain = [('company_id', '=', rec.company_id.id), ('l10n_latam_use_documents', '=', True), ('type', '=', 'sale')]
             journal = self.env['account.journal']
-            if res_code in ['8', '9', '10'] and rec.journal_id.l10n_ar_afip_pos_system not in expo_journals:
```

Comment:
I removed the 8 because it was old code that we removed and now does not exist.

NOTE: 8 and 9 are now one afip resposibility represented with '9' (Foreign Customer/Vendor)

Reply:
Hi @zaoral 
I tried to test it creating a new company in the data base but I didn't  see the warning/error.  Here is the process:
 https://drive.google.com/file/d/1_8TLDdEUzaF_xkLAKpuAwQMZ34ZlTU70/view

Reply:
Hi @ren-odoo 
Thank you for the feedback.

Mujum, I will reviewed
I tested locally and it was working but I tested with EDI installed.
I will try to test it without and let you know.

Also, I have review this with Juan and we will like made somes changes to the code we think will be good.

Let you know when this PR is ready in order to be reviewed again.

Best Regards